generators/app/index.js

```diff
@@ -590,15 +641,7 @@ module.exports = class extends Generator {
         appName: this.packageName,
         dependencies: this.dependencies,
         port: this.httpPort,
-        enableHTTPS: this.enableHTTPS,
-        httpsOnly: this.httpsOnly,
-        httpsPort: this.httpsPort,
-        pfx: this.pfx,
-        keyPath: this.keyPath,
-        passphrase: this.passphrase,
-        ca: this.ca,
-        requestCert: this.requestCert,
-        rejectUnauthorized: this.rejectUnauthorized,
+        https: this.httpsParams,
```

Comment:
when creating a fresh app, there are still issues:

```
⚠️   Missing script "dev-install"!
⚠️   Missing script "dev-prune"!
```

config still needs the latest scripts https://github.com/rooseveltframework/roosevelt/blob/master/lib/defaults/scripts.json

(not relevant to this line, just dropping the comment here to have somewhere to put it)

Reply:
Fixed with 9ea6f32